### PR TITLE
EUI-4487 my-access-specific access aprroved count

### PR DESCRIPTION
**Pull request by tarikdogan-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4487

### Change description ###

Display showing total number of new case role assignments retrieved

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/work-allocation-2/containers/task-home/task-home.component.ts [hunk 1]
```diff
@@ -3,8 +3,9 @@ import { ActivatedRouteSnapshot, NavigationEnd, Router, RoutesRecognized } from
 import { FilterPersistence, SubNavigation } from '@hmcts/rpx-xui-common-lib';
 import { Observable, Subscription } from 'rxjs';
 import { of } from 'rxjs/internal/observable/of';
+import { AllocateRoleService } from 'src/role-access/services';
```
Comment (connorpgpmcelroy) [inline src/work-allocation-2/containers/task-home/task-home.component.ts:6]: Import includes src

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Minor change needed

File: api/roleAccess/index.ts [hunk 2]
```diff
@@ -93,6 +93,16 @@ export async function getJudicialUsers(req: EnhancedRequest, res: Response, next
   }
 }
 
+export async function getSpecificAccessApproved(req, resp, next) {
+  try {
+    const currentUserAssignments = (req.session.roleAssignmentResponse as RoleAssignment[]);
```
Comment (udaydenduluri33) [inline api/roleAccess/index.ts:98]: Add safety check `if(req.session && req.session.roleAssignmentResponse)...`

File: src/role-access/services/allocate-role.service.ts [hunk 3]
```diff
@@ -75,6 +75,10 @@ export class AllocateRoleService {
     return this.http.post<CaseRole[]>(`${AllocateRoleService.roleUrl}/access-get`, {caseId, jurisdiction, caseType, assignmentId});
   }
 
+  public getSpecificAccessApproved(): Observable<any> {
```
Comment (udaydenduluri33) [inline src/role-access/services/allocate-role.service.ts:78]: please make it strongly typed instead of any
`Observable<someType...>`

**Review by udaydenduluri33 — APPROVED** [review]
LGTM @tarikdogan-hmcts thanks

**Review by connorpgpmcelroy — APPROVED** [review]
Committed a reordering of the import for the comment I had - LGTM now

**Pull request merged**